### PR TITLE
Prep template extension for release

### PR DESCRIPTION
**Pull request by jamarw**
1. Update readme, changelog and extension version.

Includes changes from PR [154](https://github.com/Microsoft/vscode-docs-authoring/pull/154)

### CODE REVIEW

File: docs-article-templates/README.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ For the first release of the Docs Article Templates extension, only simple Markd
 
 ![apply template](https://github.com/Microsoft/vscode-docs-authoring/raw/master/media/video/apply-template.gif)
 
+## To create a Learn module in VS Code
+
+1. In VSC open to the root working folder of your repository containing the achievements, path, and "product" folders.
+1. If you don't have the Docs Markdown extension installed, hit F1 to open the command palette, start typing "template" to filter, then click `Docs: Template`. If you do have Docs Markdown installed, you can use either the command palette or click `Alt+M` to bring up the Docs Markdown QuickPick menu, then select Template from the list.
+1. From the drop-down list of templates select `Learn module`.
+1. SWhen prompted select the module's parent folder.
```
Comment (heskew) [inline docs-article-templates/README.md]: `/s/SWhen/When/`

File: docs-article-templates/README.md [hunk 2]
```diff
@@ -12,6 +12,17 @@ For the first release of the Docs Article Templates extension, only simple Markd
...
+1. EIn the "Enter module name" prompt enter your module's name. This is used to create the module, media, and include folders, the index.yml, and the module uid. Example: Module with the friendly name `What is m365` will automatically be formatted as `what-is-m365` by the template extension for the applicable fields.
```
Comment (heskew) [inline docs-article-templates/README.md]: `s/EIn/In/`

File: docs-article-templates/src/helper/module-builder.ts [hunk 3]
```diff
@@ -82,29 +84,46 @@ export function createModuleDirectory() {
 // data used to create the module yml file.
 // check settings.json for repo value.  if there's no value, the root path directory will be considered the repo name.
 export function updateModule(units) {
-    if (learnRepoId) {
-        learnRepo = learnRepoId;
-    } else {
+    if (!learnRepoId) {
```
Comment (heskew) [inline docs-article-templates/src/helper/module-builder.ts]: Nit, and sorry if this is against current practices in here: I have a real aversion to coercing strings to bools. Would it be possible to have these more explicit, inline or some `!isEmtpy(str)` helper (homegrown, lodash, etc)?

File: docs-article-templates/src/helper/module-builder.ts [hunk 4]
```diff
@@ -82,29 +84,46 @@ export function createModuleDirectory() {
...
         learnRepo = repoName;
     }
+    if (!gitHubID) {
+        author = `...`;
```
Comment (heskew) [inline docs-article-templates/src/helper/module-builder.ts]: Nit: `` ` `` isn't necessary

Comment (heskew) [reply docs-article-templates/src/helper/module-builder.ts]: But seems it's consistent so 👍 ? :)

File: docs-article-templates/README.md (hunk 1 above)
Comment (jamarw) [reply docs-article-templates/README.md]: Nice catch, copy and paste fun.

File: docs-article-templates/README.md (hunk 2 above)
Comment (jamarw) [reply docs-article-templates/README.md]: Nice catch, copy and paste fun.

File: docs-article-templates/src/helper/module-builder.ts (hunk 3 above)
Comment (jamarw) [reply docs-article-templates/src/helper/module-builder.ts]: Good call, creating a task for this so we can capture all the string checks.

File: docs-article-templates/src/helper/module-builder.ts (hunk 4 above)
Comment (jamarw) [reply docs-article-templates/src/helper/module-builder.ts]: You're right, I'll add these to the cleanup task.

**Pull request merged**